ansible/configs/ocp-clientvm/env_vars.yml

```diff
@@ -177,3 +180,31 @@ instances:
       - device_name: "{{docker_device}}"
```

Comment:
Can we consider dropping this extra device and making the size of the root disk like 30GB? We don't need this much storage on a client VM for anything and we have finite amounts of storage available.

Reply:
I'm happy to reduce this, but i think it's out of the scope of this PR.

i created: https://github.com/redhat-cop/agnosticd/issues/882